Maps.md

```diff
@@ -0,0 +1,25 @@
+This is a general guide about Maps for all FullScreenShenanigans projects. This guide will explain the concept of Maps, Areas, and Locations within a game and how they're connected.
+
+### Table of Contents
+1. [MapsCreatr](#mapscreatr)
+2. [AreaSpawnr](#areaspawnr)
+
+# MapsCreatr
+
+MapsCreatr is a module that handles the storage of Maps, Areas, and Locations. 
+
+A Map is a collection of Areas and an Area is a space with predefined boundaries and Things. Locations are entry points within an Area, where an entry point is where the player ends up after transitioning to a new Area.
+
+Generally in a FullScreenShenanigans project, each Map information is stored in its own .js file, outlining all Areas and Locations within that map. When the game is started, MapsCreatr loads all this data, goes through every Map and uses ObjectMakr to create a new Map object, and finally stores it for later reference.
+
+More can be read about MapsCreatr on its [Readme](https://github.com/FullScreenShenanigans/MapsCreatr/blob/master/README.md).
+
+# AreaSpawnr
+
+AreaSpawnr is a module that works with MapsCreatr to spawn Things in a certain Area.
+
+When a new Area is entered by the player, AreaSpawnr uses the information from that Area's Map, stored in MapsCreatr, to spawn that Area's predefined Things (or PreThings) in the Area. 
+
+## PreThings
+
+PreThings are just the default Things that are specific to an Area that are spawned upon entering. They are specified in a Map's .js file as raw JSON data and are interpreted by MapsCreatr's `analyzePreSwitch` method to determine what kind of PreThing they are (raw PreThing, location setter, or macro). When the Map data is loaded, every PreThing is created using ObjectMakr and stored for later reference.
```

Comment:
This entire paragraph is inaccurate :(

* I don't understand the first one. See PreThing.ts in MapsCreatr.
* If you say "described" instead of "specified" it would be more accurate.
* I mentioned analyzePreSwitch because it's a good starting place to understand the code, not because it's a good thing to mention here. IMO it's not.
* A PreThing only ever stores a Thing. JSON commands can specify things other than PreThings (specifically just macros) but those will always result in >=0 PreThings.
    * Where are location setters mentioned? That's deprecated behavior, could you please file an issue in whatever repo to remove that documentation?
* "Every PreThing is created using ObjectMakr" no, they already exist. Saying they're made with ObjectMakr is vague and misleading. See AreaSpawnr's methods.